static/js/containers/TermsOfServicePage.js

```diff
@@ -2,8 +2,13 @@
 import React from 'react';
 import Button from 'react-bootstrap/lib/Button';
 import { connect } from 'react-redux';
+import Grid, { Cell } from 'react-mdl/lib/Grid';
+import UserImage from '../components/UserImage';
 
 import { saveProfile, FETCH_SUCCESS } from '../actions';
+import {
+  DASHBOARD_COURSE_HEIGHT,
+} from '../constants';
```

Comment:
this is the right height to use, but it seems weird to me to reuse the style in this context. This isn't the dashboard. This isn't a course.